x-pack/plugin/esql/qa/testFixtures/src/main/resources/date.csv-spec

```diff
@@ -614,3 +614,60 @@ birth_date:datetime
 1952-02-27T00:00:00.000Z
 1953-04-21T00:00:00.000Z
 ;
+
+dateMathAddDatePeriodsFromLeft
+row n = to_dt("1953-04-04T00:00:00.000Z") | eval then = 4 years + 3 months + 2 weeks + 1 day + n | keep then;
```

Comment:
Add also 1 year + 1 year  + 2 years, 1 months + 1 week + 1 month+ + 1 week , etc...
Mix the types to make sure they properly get combined.

Reply:
The unit tests should already cover a lot of ground there, but I'll happily add CSV tests as well.